editor.planx.uk/src/pages/Teams.tsx

```diff
@@ -1,71 +1,102 @@
-import Edit from "@mui/icons-material/Edit";
-import Visibility from "@mui/icons-material/Visibility";
 import Box from "@mui/material/Box";
 import Card from "@mui/material/Card";
+import Container from "@mui/material/Container";
 import { styled } from "@mui/material/styles";
 import Typography from "@mui/material/Typography";
 import { Team } from "@opensystemslab/planx-core/types";
 import React from "react";
 import { Link } from "react-navi";
+import Dashboard from "ui/editor/Dashboard";
 
 import { useStore } from "./FlowEditor/lib/store";
 
+interface TeamTheme {
+  slug: string;
+  primaryColour: string;
+}
+
 interface Props {
   teams: Array<Team>;
+  teamTheme: Array<TeamTheme>;
 }
 
-const Root = styled(Box)(({ theme }) => ({
-  backgroundColor: theme.palette.background.dark,
-  color: "#fff",
+export const Root = styled(Box)(({ theme }) => ({
+  backgroundColor: theme.palette.background.default,
   width: "100%",
-  flex: 1,
+  display: "flex",
+  alignItems: "flex-start",
+  flexGrow: 1,
+}));
+
+const StyledLink = styled(Link)(() => ({
+  textDecoration: "none",
+}));
+
+const TeamCard = styled(Card)(({ theme }) => ({
+  display: "flex",
   justifyContent: "flex-start",
   alignItems: "center",
+  marginBottom: theme.spacing(2),
+  color: theme.palette.text.primary,
+  outline: `1px solid ${theme.palette.border.light}`,
+  outlineOffset: "-1px",
+  borderRadius: "1px",
 }));
 
-const Dashboard = styled(Box)(({ theme }) => ({
-  backgroundColor: theme.palette.background.dark,
-  color: "#fff",
-  width: "100%",
-  maxWidth: 600,
-  margin: "auto",
-  padding: theme.spacing(8, 0, 4, 0),
+const TeamColourBand = styled(Box)(({ theme }) => ({
+  display: "flex",
+  alignSelf: "stretch",
+  width: theme.spacing(1.5),
+  zIndex: 1,
 }));
 
-const StyledLink = styled(Link)(() => ({
-  textDecoration: "none",
-}));
+const Teams: React.FC<Props> = ({ teams, teamTheme }) => {
+  const canUserEditTeam = useStore.getState().canUserEditTeam;
 
-const Teams: React.FC<Props> = ({ teams }) => {
+  const editableTeams = teams.filter((team) => canUserEditTeam(team.slug));
+  const viewOnlyTeams = teams.filter((team) => !canUserEditTeam(team.slug));
+
+  const renderTeams = (teamsToRender: Array<Team>) =>
+    teamsToRender.map(({ name, slug }) => {
+      const theme = teamTheme.find((t) => t.slug === slug);
+      const primaryColour = theme ? theme.primaryColour : "#000";
```

Comment:
`team_theme.primary_colour` is a required field (with a default value) in the database so we should be fine to not have a default here. 

It's possible that this is required to get around some TypeScript issues, but this should be resolved by improving the query and dropping the `find()` (see previous comment).

Reply:
I'm now calling this as `team.theme.primaryColour` without a default value